rag_experiment_accelerator/ingest_data/acs_ingest.py

```diff
@@ -125,7 +125,7 @@ def generate_qna(environment, config, docs, azure_oai_deployment_name):
             new_df = new_df._append(data, ignore_index=True)
```

Comment:
**[Severity: Medium]** The use of `_append()` on the DataFrame is generally discouraged as it is a private method. Instead, consider using `pd.concat()` to append data to the DataFrame. This will make the code more maintainable and future-proof.


---

**[peer-review-pal]** We are sorry for the intrusion. Feel free to ignore or delete the comments/issue with the tag - they are automatically generated by our [Hack project](https://hackbox.microsoft.com/hackathons/hackathon2024/project/81776) - repository **[Peer Review Pal](https://github.com/microsoft/peer-review-pal)**.
If you find them useful, want us to delete them, have any feedback at all or want to contribute - reach out to some of us: juliame@microsoft.com vadimkirilin@microsoft.com oteslenko@microsoft.com vimeena@microsoft.com
You can prevent future comments from appearing by adding **#prp-ignore** to your PR description.